Source/WebKit/UIProcess/ProcessThrottler.h

```diff
@@ -122,20 +111,19 @@ class ProcessThrottler : public CanMakeWeakPtr<ProcessThrottler> {
     ~ProcessThrottler();
 
     using Activity = ProcessThrottlerActivity;
-    using ActivityVariant = std::variant<std::nullptr_t, UniqueRef<Activity>>;
 
     void ref() const;
     void deref() const;
 
     using ForegroundActivity = Activity;
-    UniqueRef<Activity> foregroundActivity(ASCIILiteral name);
+    Ref<Activity> foregroundActivity(ASCIILiteral name);
 
     using BackgroundActivity = Activity;
-    UniqueRef<Activity> backgroundActivity(ASCIILiteral name);
-    UniqueRef<Activity> quietBackgroundActivity(ASCIILiteral name);
+    Ref<Activity> backgroundActivity(ASCIILiteral name);
+    Ref<Activity> quietBackgroundActivity(ASCIILiteral name);
 
-    static bool isValidBackgroundActivity(const ActivityVariant&);
-    static bool isValidForegroundActivity(const ActivityVariant&);
+    static bool isValidBackgroundActivity(RefPtr<Activity>);
```

Comment:
Should take a `const Activity&` or a `const Activity*`, not a smart pointer. We're causing unnecessary ref-counting churn.